### ISSUE TITLE
Thesisdown knit inconsistently incorporating new changes to PDF output

### ISSUE THREAD

**Issue by rachelledig** [ismayc/thesisdown#165]
Hello!

I am having issues with my thesisdown template not incorporating new changes I have made to individual chapter documents (and prelims) to my final output format. It seems to knit fine, with some warnings about citations (my issue) but basically, I will make a change to a given file, save it, knit, and the output document (in this case PDF) does not have any of my new changes in it.

Sometimes deleting the old PDF document (and subsequent tex output) will solve the issue. Sometimes I need to quit and restart the R project I am working in, knit again, and the changes appear. Sometimes nothing works. There seems to be no sense to the madness. 

Basically, I would like my changes to be incorporated every time I knit. 

Is there a temporary cache file somewhere that is causing this? any tips or tricks?

 - OS: Mac Catalina 10.15.7
 - R version 4.0.3
 - thesisdown 0.2.0
